test/test_images/utils.go

```diff
@@ -56,23 +56,17 @@ const (
 	ConfigLoggingEnv = "K_CONFIG_LOGGING"
 )
 
-// ConfigureTracing can be used in test-images to configure tracing
+// ConfigureTracing can be used in test-images to configure tracing.
 func ConfigureTracing(logger *zap.SugaredLogger, serviceName string) error {
 	tracingEnv := os.Getenv(ConfigTracingEnv)
-
-	if tracingEnv == "" {
-		return tracing.SetupStaticPublishing(logger, serviceName, config.NoopConfig())
-	}
-
 	conf, err := config.JSONToTracingConfig(tracingEnv)
 	if err != nil {
-		return err
+		logger.Warn("Error while trying to read the tracing config, using NoopConfig: ", err)
 	}
-
 	return tracing.SetupStaticPublishing(logger, serviceName, conf)
```

Comment:
if `conf` is `nil` line 66 panics.

can we do the other way around when `loggingEnv` in `ConfigureLogging` is `""` we fall back to a logger created with `zap.NewDevelopment()`?

Reply:
The point is that `conf` should never be nil. becaue JSONToTracingConfig always returns a Noop config. 
We can do it the other way round. Just for my understanding - why do you prefer that?

Reply:
> The point is that conf should never be nil. because JSONToTracingConfig always returns a Noop config.

Ok, then I'm fine with this too

/lgtm
/approve

Reply:
Thanks :-)